modules/nw-modifying-operator-install-config.adoc

```diff
@@ -108,7 +114,7 @@ specify only the parameters that you want to change.
 installation program deletes the `manifests/` directory when creating the
 cluster.
 
-ifdef::vsphere[]
+ifdef::vsphere,vmc[]
```

Comment:
I don't think I'm seeing this here: https://deploy-preview-29632--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-vmc-network-customizations-user-infra.html#modifying-nwoperator-config-startup_installing-vmvc-network-customizations-user-infra

Reply:
Caused by a typo in the `:context:`. This also caused several of the other rendering issues. Fixed!